web-api/openapi/front_office.yaml

```diff
@@ -48,20 +48,51 @@ security:
 
 paths:
   /users/{user_id}/personalized_documents:
-    get:
+    parameters:
+      - $ref: './parameters/path/id.yml#/UserId'
+
+    post:
```

Comment:
One interesting observation:

We always "brake" the http standard, either it's a post request which semantically shouldn't be one or it's a get request with a body (which given the standard should have no semantic meaning (but must still be passed on)).

The question is if we shouldn't go with get + body (like e.g. ES).

In the past (many many years ago) some (rare) middle ware messed with that but by now this shouldn't matter and the standard had been updated to put it into a "not stylistically recommended but okay" situation AFIK. 

Main benefit would be that browser caching get requests might still work with that :thinking: .

Reply:
The problem that I see is also with that the API from the browser seems not to allow adding body to GET.
https://xhr.spec.whatwg.org/#the-send()-method
https://fetch.spec.whatwg.org/#dom-request (point 35).

This endpoint is not dissimilar from `/semantic_search` and we already use POST there.